renovate.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base"
+  ],
+  "labels": ["dependencies"],
+  "packageRules": [
+    {
+      "matchUpdateTypes": ["minor", "patch"],
+      "excludePackagePatterns": ["sarama"],
+      "automergeType": "branch",
+      "matchCurrentVersion": "!/^0/",
+      "automerge": true
```

Comment:
What happens if a test fails?

I mean, if renovate create a branch, push the changes, and a test does not pass. Will renovate stuck there or will open a PR for us to see that?

The question is because I think we are afraid of updating things. I would update Sarama and any 0ver without issues because I trust our tests. But only if we have a mechanism that warns us that there is an error.

To be honest, I do not know how to reproduce this kind of error I am pointing here... Forking from a version before Sarama broke?

Reply:
If the tests pass, the branch gets merged after some time.
If the tests don't pass, Renovate will create a PR.

Reply:
> I wonder if it would be possible to include an additional rule so majors and Sarama so the corresponding updates are not auto-merged but renovate opens a PR instead.

After our latest conversation, I understand it is already like that. Thank you for all the explanations @kang-makes, @alvarocabanas